packages/kit/src/plugin.ts

```diff
@@ -35,7 +33,9 @@ export function normalizePlugin (plugin: NuxtPlugin | string): NuxtPlugin {
 
   if (!existsSync(plugin.src) && isAbsolute(plugin.src)) {
     try {
-      plugin.src = resolvePathSync(isWindows ? pathToFileURL(plugin.src).href : plugin.src, { extensions: tryUseNuxt()?.options.extensions })
+      plugin.src = normalize(resolveModulePath(plugin.src, {
+        extensions: tryUseNuxt()?.options.extensions ?? ['.js', '.mjs', '.cjs', '.ts', '.tsx', '.mts', '.cts'],
```

Comment:
we are supporting `tsx`, there is also `mtsx` and `ctsx` (crying out loud!), ts and jiti also support them. https://github.com/unjs/jiti/blob/main/src/options.ts#L13-L23